### ISSUE TITLE
Conditions Victory

### ISSUE THREAD

**Issue by VikLeroy** [alephtwo/tragedy-looper-generator#30]
Is it possible to add "Conditions Victory" in the  Cheatsheet ?

For example: 
If "Light of the Avenger" is picked, telle that if 2+ Intrigue is on brain location, loops ends
If there is a key person and she can be killed by suicide (trigger event), or by a serial killer
etc.

**Comment by alephtwo** [comment]
@VikLeroy Yep! I believe an earlier iteration of the design had `winCondition` as a property on a variety of the types involved, but it got very involved when trying to combine them. It's definitely something I would like to add.

**Comment by alephtwo** [comment]
I added really primitive support for this in 50e68070172db03ff12edf48fc117f3e1a6a961f. It's nothing fancy but it should do the trick. 

I added a `winCondition` flag to each type that could possibly end the loop, and flagged them accordingly. Note that it doesn't know about combos, just the actual thing that causes the protagonists to lose - and that's probably fine. 

For example: 
![image](https://user-images.githubusercontent.com/3613974/172735931-313a4a6d-9b12-4b07-8250-33d161b5b04d.png)

Give it a shot at the live instance and let me know what you think: https://alephtwo.github.io/tragedy-looper-generator/
